assets/src/dashboard/style.css

```diff
@@ -75,7 +74,3 @@ body.web-story_page_stories-dashboard #wpfooter {
 button {
   font-size: inherit;
 }
-
```

Comment:
Why was this removed?

Reply:
Not removed per se, but I restructured the CSS a bit. If you look at Percy the visual outcome should be the same.